src/app/scopes-dialog/scopes-dialog.component.ts

```diff
@@ -8,9 +8,6 @@ import { IPermissionScope } from '../base';
 import { GraphExplorerComponent } from '../GraphExplorerComponent';
 import { PermissionScopes } from './scopes';
 
-declare let fabric;
```

Comment:
Dropping fabric and MWF. What is the trade off?

Reply:
The fabric dialog does not trap focus. Which makes it hard to make the modal accessible.  The `<dialog>` tag traps focus out of the box.

Reply:
Did you take a look at this article referenced by Melanie for trapping focus: https://www.marcozehe.de/2015/02/05/advanced-aria-tip-2-accessible-modal-dialogs/

Why won't this work with our current UI elements?